DOc/docs/source/Options/user-story-1.rst

```diff
@@ -37,7 +37,7 @@ are provided.
 
 To illustrate the process of finding relevant information, the location
 of the data and the accountable person, the following scenario is
-described: A data engineer is working in an HR project investigating the
+described: A data engineer is working on a HR project investigating the
```

Comment:
@gabypascha "an" HR is correct :) 
https://www.google.com/search?q=an+hr+or+a+hr&rlz=1C1VDKB_nlNL1013NL1014&oq=an+HR+&aqs=chrome.0.0i512j69i57j0i512l4j69i60l2.3422j0j7&sourceid=chrome&ie=UTF-8

Reply:
Better?